vnext/Scripts/Microsoft.ChakraCore.ARM64.nuspec

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+  Sample usage:
+
+nuget pack Microsoft.ChakraCore.ARM64.nuspec `
+  -Properties "id=ReactWindows.ChakraCore.ARM64;targetRoot=D:\chakra;repoUrl=https://github.com/microsoft/ChakraCore;repoBranch=master;repoCommit=$(git rev-parse HEAD)" `
+  -Version 1.11.20.0
+ -->
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <!-- i.e.: Microsoft.ChakraCore.vc140.ARM64 -->
+    <id>$id$</id>
+    <version>$version$</version>
+    <authors>Microsoft</authors>
+    <owners>Chakra Team</owners>
```

Comment:
should this say Microsoft instead?

Reply:
Not sure (I'm not against).

I took the value straight from ChakraCore's own NUSPECs.
Technically, they produced the binaries. We're just packaging them.

We should have a discussion to set a clear guidance for dependencies we package ourselves.

Reply:
Why are we packaging dependencies that come from nuget anyway? Are we forked?

Reply:
No. This new package contains only the ARM64 build of ChakraCore.

Apparently, there will be no more "breaking" updates to the original NuGet, including an already supported platform.